.github/workflows/live-tests.yml

```diff
@@ -64,29 +38,3 @@ jobs:
         uses: ./.github/actions/live-tests-shared
```

Comment:
Up to you but if we will only have single job here then no reasons to extract steps into shared actions file.

Reply:
hm ok, but maybe we need different versions again in the future, so let's leave it as is